ethsigner/signer/multikey/src/main/java/tech/pegasys/ethsigner/signer/multikey/SigningMetadataTomlConfigLoader.java

```diff
@@ -88,8 +88,9 @@
       final TomlParseResult result =
           TomlConfigFileParser.loadConfigurationFromFile(file.toAbsolutePath().toString());
 
-      final Optional<ThrowingTomlTable> signingTable =
+      final Optional<TomlTable> tomlTable =
           getSigningTableFrom(file.getFileName().toString(), result);
+      final Optional<ThrowingTomlTable> signingTable = getThrowingSigningTableFrom(tomlTable);
```

Comment:
Thinking the ThrowingTomlTable adapter should be renamed and just have an additional method that doesn't throw so we don't need to switch between the classes to use.

Reply:
Thats what I originally had. Removed the "optional" methods out if it.

Reply:
refactored.